src/bindgen.rs

```diff
@@ -36,3 +37,16 @@ pub fn wasm_bindgen_build(path: &str, name: &str) {
         .unwrap_or_else(|e| panic!("{} failed to execute process: {}", emoji::ERROR, e));
     pb.finish();
 }
+
+fn wasm_bindgen_installed() -> bool {
+    if let Ok(path) = env::var("PATH") {
+        return path.split(":")
```

Comment:
Since PATH is just a String you can test by doing path.contains("wasm-bindgen")

Reply:
You also don't need to use return just leave off the semicolon! :D 
Rust will always return a value for the last expression without a semicolon. We use return when we want to exit a function early much like you did in the file above!

Reply:
Oh you know what you're right. PATH contains directories of binaries that can be run not the binaries themselves! You'll need to traverse them to find it. I'd like so say we can assume it's in .cargo/bin and just check there, which is easier. However, I feel we should probably check each one like you've done already.

Regarding cross platform you can do [conditional](https://doc.rust-lang.org/book/first-edition/conditional-compilation.html) compilation depending on the platform so we can set a constant in the file and use it here to say "on this platform use : and on this one use" whatever it is.